### PR TITLE
[wasm][debugger] Shipping internal package for webcil

### PR DESCRIPTION
**Pull request by thaystg**
Shipping internally webcil package that will be consumed from an external repo.

### CODE REVIEW

**Comment by thaystg** [comment]
@akoeplinger is this enough to ship the package internally as we do for `Microsoft.NETCore.BrowserDebugHost.Transport`?

File: src/mono/nuget/mono-packages.proj [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <ProjectReference Include="Microsoft.NET.Runtime.WebAssembly.Sdk\Microsoft.NET.Runtime.WebAssembly.Sdk.pkgproj" />
     <ProjectReference Include="..\wasm\templates\Microsoft.NET.Runtime.WebAssembly.Templates.csproj" />
     <ProjectReference Include="Microsoft.NET.Sdk.WebAssembly.Pack\Microsoft.NET.Sdk.WebAssembly.Pack.pkgproj" />
-
+    <ProjectReference Include="Microsoft.NET.WebAssembly.Webcil\Microsoft.NET.WebAssembly.Webcil.pkgproj" />
```
Comment (akoeplinger) [inline src/mono/nuget/mono-packages.proj:12]: This pkgproj doesn't exist, you'd need to create it

File: src/tasks/Microsoft.NET.WebAssembly.Webcil/Microsoft.NET.WebAssembly.Webcil.csproj [hunk 2]
```diff
@@ -9,8 +9,7 @@
     <Serviceable>true</Serviceable>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsShipping>true</IsShipping>
-    <!-- this assembly should not produce a public package, rather it's meant to be shipped by the
-          WasmAppBuilder task and the BrowserDebugProxy -->
+    <!-- this assembly should produce a non-shipping package, because it will be used by BrowserDebugProxy -->
```
Comment (akoeplinger) [inline src/tasks/Microsoft.NET.WebAssembly.Webcil/Microsoft.NET.WebAssembly.Webcil.csproj:12]: WasmAppBuilder still uses this though?

**Review by akoeplinger — APPROVED** [review]

File: src/mono/nuget/mono-packages.proj (hunk 1 above)
Comment (thaystg) [reply src/mono/nuget/mono-packages.proj:12]: Oh, I forgot to add the file!

File: src/tasks/Microsoft.NET.WebAssembly.Webcil/Microsoft.NET.WebAssembly.Webcil.csproj (hunk 2 above)
Comment (thaystg) [reply src/tasks/Microsoft.NET.WebAssembly.Webcil/Microsoft.NET.WebAssembly.Webcil.csproj:12]: Yes.

**Comment by thaystg** [comment]
> I'm not sure what the goal here is and what exactly you mean with "internal". The distinction we have is Shipping (i.e. to nuget.org) or Non-Shipping (to some dnceng-feeds) but both of these still mean the package is publicly available.

The goal is to use it in another repo. It's okay to be in dnceng-feeds. Thanks for the explanation.

File: src/mono/nuget/mono-packages.proj (hunk 1 above)
Comment (akoeplinger) [reply src/mono/nuget/mono-packages.proj:12]: @thaystg if all you need is a nuget package then you don't need the .pkgproj (which is kinda deprecated), you can just set `<IsPackable>true</IsPackable>` in the .csproj.

**Review by akoeplinger — APPROVED** [review]

**Pull request merged**